index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header>
+      <nav id="navbar">
+        <div class="links-container">
+          <a href="#">About</a>
+          <a href="#" id="work">Work</a>
+          <a href="#">Contact</a>
+        </div>
```

Comment:
An unordered list can serve a better purpose for us to achieve the same in a much better way. Implement this with a a <ul> tag